projects/core/src/cart/bundle/occ/adapters/default-occ-bundle-config.ts

```diff
@@ -0,0 +1,13 @@
+import { OccConfig } from 'projects/core/src/occ/config/occ-config';
+
+export const defaultOccBundleConfig: OccConfig = {
+  backend: {
+    occ: {
+      endpoints: {
+        bundleStart: 'users/${userId}/carts/${cartId}/bundles?fields=DEFAULT',
```

Comment:
Ideally we should avoid `DEFAULT` in the endpoints configurations. Can we replace it with specific fields?